src/coreclr/jit/gentree.h

```diff
@@ -6416,47 +6416,48 @@ struct GenTreeJitIntrinsic : public GenTreeMultiOp
 
     var_types GetAuxiliaryType() const;
 
-    CorInfoType GetSimdBaseJitType() const
+    var_types GetNormalizedSimdBaseType() const
```

Comment:
This should be unnecessary now. `TYP_I_IMPL` is just a `#define` that maps to `TYP_LONG` or `TYP_INT` already.